.licenserc.yaml

```diff
@@ -0,0 +1,40 @@
+# Licensed to Apache Software Foundation (ASF) under one or more contributor
+# license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright
+# ownership. Apache Software Foundation (ASF) licenses this file to you under
+# the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+header:
+  license:
+    spdx-id: Apache-2.0
+    copyright-owner: Apache Software Foundation
+
+  paths-ignore:
+    - NOTICE
+    - LICENSE
+    - RELEASE_NOTES
+    - DISCLAIMER
+    - mvnw.cmd
+    - .mvn
+    - .gitattributes
+    - '**/*.md'
+    - '**/*.json'
+    - '**/*.iml'
+    - '**/*.input'
+    - '**/*.lock'
+    - '**/*.ini'
+    - '**/.gitignore'
+    - '**/org/apache/wayang/commons/util/profiledb/**'
+
```

Comment:
It is best to modify all files under this package to jar dependencies

Reply:
Because we are using Apache Rat to check the licenses, and the header in that file did not throw any exception because have the elements that Apache Rat Check, however, that does means that is a correct one, if you have a better one or a recomendation we can change it

Reply:
My suggestion here is to modify it as a jar dependency, (if possible)
It is important to keep NOTICE as brief and simple as possible, as each addition places a burden on downstream consumers.

Here are some introductions, you can take a look:https://github.com/apache/skywalking-eyes